staking/tests/position_lifecycle.ts

```diff
@@ -329,6 +321,15 @@ describe("position_lifecycle", async () => {
       })
       .rpc();
 
+    // Positions have swapped so this should work
+    await program.methods
+      .closePosition(0, PythBalance.fromString("50").toBN(), votingProduct)
+      .accounts({
+        targetAccount: votingProductMetadataAccount,
+        stakeAccountPositions: stakeAccountAddress,
+      })
+      .rpc();
+
```

Comment:
We should download the position account and confirm that all the positions are what we expect